docs/user_guide/index.md

```diff
@@ -0,0 +1,25 @@
+---
+myst:
+  html_meta:
+    "description lang=en": |
+      Documentation for beginner users.
+---
+# User Guide
+
+```{toctree}
+:caption: Installing SED
+:maxdepth: 1
+installation
+```
+
+```{toctree}
+:caption: Basic concepts
+../tutorial/1_binning_fake_data
+../tutorial/8_jittering_tutorial
+../tutorial/3_metadata_collection_and_export_to_NeXus
```

Comment:
I don't find these helpful here in this order. They hardly can be understood here without the other tutorials.

Reply:
I get the point. But I think we need to include more documentation about the library and the workflow tutorials are for specific cases.
For example, I wouldn't use the WSe2 entire dataset for jittering example but we can take a small subset of it and show how jittering works similar to tutorial 1.

Reply:
Restructured